solar/solar/dblayer/__init__.py

```diff
@@ -1,6 +1,10 @@
 from solar.dblayer.model import ModelMeta
 from solar.dblayer.riak_client import RiakClient
+
 client = RiakClient(protocol='pbc', host='10.0.0.2', pb_port=8087)
 # client = RiakClient(protocol='http', host='10.0.0.2', http_port=8098)
 
 ModelMeta.setup(client)
+
+from solar.dblayer import standalone_session_wrapper
+standalone_session_wrapper.create_all()
```

Comment:
Because you need to import model (directly or indirectly) it when you use solar, so adding it there adds it automaticaly "everywhere". It also checks executable so it's enabled only for standalone python executions.